packages/odyssey-design-tokens/src/font.json

```diff
@@ -1,5 +1,28 @@
 {
   "font": {
+    "color": {
+      "body": {
+        "comment": "Base text color used for most typography",
+        "value": "{hue.neutral.900.value}"
+      },
+      "heading": {
+        "comment": "Base text color used for headings",
+        "value": "{hue.neutral.900.value}"
+      },
+      "bodyInverse": {
+        "comment": "Inverse text color typically used on a colorful background",
+        "value": "{hue.neutral.white.value}"
+      },
+      "primary": {
+        "comment": "Branded text color, primarily used within button",
+        "value": "{palette.primary.main.value}"
+      },
+      "sub": {
+        "comment": "Text color for subordinate text like captions",
+        "value": "{hue.neutral.600.value}"
+      },
+      "danger": { "value": "{palette.danger.main.value}" }
+    },
```

Comment:
Also flagging: it's incredibly likely that we add another token for text at 700. Need to find a way to describe use case(s) to help with tokenization

Reply:
Ok, will revert to danger and action to cover more use cases: 
body
bodyInverse
danger
action

Can caption/sub/secondary be 'secondary'? There are so many varied uses, it needs to be broad...